examples/sparsestream/server/util.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+
+
+def get_tokens():
+
+    with open("./server/config.yaml") as file:
+        config = yaml.safe_load(file.read())
+
+    return config["twitter_tokens"]
```

Comment:
this function could be in server.py, also could we add the config path as an argument?